readme.md

```diff
@@ -86,6 +86,22 @@ Run `np` without arguments to launch the interactive UI that guides you through
 <img src="screenshot-ui.png" width="1290">
 
 
+## Config
+
+`np` can be configured both locally and globally. When using the global `np` binary, you can configure any of the CLI flags in either a `.np-config.js` or `.np-config.json` file in the home directory. When using the local `np` binary, for example, in a `npm run` script, you can configure `np` by setting the flags in either a top-level `np` field in `package.json` or in a `.np-config.js` or `.np-config.json` file in the project directory.
+
+For example, this configures `np` to never use Yarn and to use `dist` as the subdirectory to publish:
+
+```js
+{
+	"yarn": false,
+	"contents": "dist"
+}
+```
```

Comment:
I think we should show what all 3 of the formats would look like.

Reply:
What do you mean? What's the 3rd format?

Reply:
1. `np` field in package.json
2. JSON file
3. JS file